src/main/webapp/WEB-INF/views/cars/list.jsp

```diff
@@ -0,0 +1,43 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Web Practice</title>
+</head>
+    <body>
+        <h1>All Cars List</h1>
+        <table>
+            <tr>
+                <td>ID</td>
+                <td>Model</td>
+                <td>Manufacturer</td>
+                <td>Drivers</td>
+            </tr>
+            <c:forEach items="${carsList}" var="car">
+                <tr>
+                    <td><c:out value="${car.id}" /></td>
+                    <td><c:out value="${car.model}" /></td>
+                    <td><c:out value="${car.manufacturer}" /></td>
+                    <td><c:out value="${car.drivers}" /></td>
+                    <td><button><a href="${pageContext.request.contextPath}/cars/delete?id=${car.id}">delete car</a></button></td>
+                    <td>
+                        <form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+                            <input type="hidden" name="car_id" value="${car.id}" />
+                            <input type="number" name="driver_id">
+                            <button type="submit">add driver by id</button>
+                        </form>
+                    </td>
+                </tr>
+            </c:forEach>
+            <form method="post" action="${pageContext.request.contextPath}/cars/add">
+                <tr>
+                    <td>#</td>
+                    <td><input type="text" name="model"></td>
+                    <td>By id: <input type="number" name="manufacturer_id"></td>
+                    <td></td>
+                    <td><button type="submit">add</button></td>
+                </tr>
+            </form>
+        </table>
```

Comment:
wow
can you provide screenshots of your pages for cars, drivers and manufacturers?

Reply:
Cars page looks pretty messy, but I suppose it would look better with some css styling or option to hide/show the `deleting` and `add driver by id` columns
![image](https://user-images.githubusercontent.com/45060278/204042522-8b51b13f-1dee-46fd-8ea2-632d622cefab.png)
![image](https://user-images.githubusercontent.com/45060278/204042529-b54da58b-bd77-4233-acb7-4fcc396b078f.png)
![image](https://user-images.githubusercontent.com/45060278/204042532-0fef03c5-b93a-4068-9423-4a647e37e4fa.png)

Reply:
looks like we forgot override `toString` for drivers
but other things are great